data/json/items/gunmod/underbarrel.json

```diff
@@ -382,6 +382,14 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 15 ] ],
     "flags": [ "STAB" ]
   },
+  {
+    "id": "makeshift_pistol_bayonet",
+    "copy-from": "pistol_bayonet",
+    "type": "GUNMOD",
+    "name": "makeshift pistol bayonet",
```

Comment:
MS pistol bayonet is meant to override the base game pistol bayonet's spot(which is then made a fab 5 recipe), direct merging doesn't make sense.